src/cr/cube/cube.py

```diff
@@ -673,7 +679,22 @@ def raw_cube_array(self):
         )
         # ---must be read-only to avoid hard-to-find bugs---
         raw_cube_array.flags.writeable = False
-        return raw_cube_array
+        return raw_cube_array.T if self._require_array_transposition else raw_cube_array
+
+    @lazyproperty
+    def _require_array_transposition(self):
+        """True of False if the raw cube array need to be transposed.
+
+        When a Cube is part of a cubeset and one of the dimension type is a NUM_ARRAY,
+        it's cube_array values have to be transposed.
+        This business rule needs to drive correctly tabbook exports when a numeric array
+        dimension is expressed.
+        """
+        return (
+            self._cube_dict.get("cube_idx") is not None
```

Comment:
Why can't we just check `self._cube_idx_arg` here?

Reply:
Cause in python logic expression like `None and True` returns None` and i'd like to be consistent with a boolean return value